### PR TITLE
Fix pattern image fill for Firefox

### PR DESCRIPTION
**Pull request by klipstein**
@basecode: using images as pattern fill is currently leading to strange artifacts in Firefox (wrong positioning of background tiles). This pull request fixes this issue.

### CODE REVIEW

File: src/renderer/svg/svg.js [hunk 1]
```diff
@@ -1219,27 +1220,29 @@ define([
...
-
-      pattern.setAttribute('width', 1 / fillRepeatX);
-      pattern.setAttribute('height', 1 / fillRepeatY);
+      pattern.setAttribute('patternUnits', 'userSpaceOnUse');
       pattern.setAttribute('x', 0);
       pattern.setAttribute('y', 0);
+      pattern.setAttribute('width', boundingBox.width / fillRepeatX);
+      pattern.setAttribute('height', boundingBox.height / fillRepeatY);
+
+      elementMatrix = attributes.matrix.clone();
```
Comment (padolsey) [inline src/renderer/svg/svg.js:1229]: Has this been tested in worker context? The real `Matrix` (with a clone method) instance will only be passed to the renderer in the iframe context ... in worker context, AFAIK, you just get a regular object.

Comment (klipstein) [inline src/renderer/svg/svg.js:1229]: @padolsey i'll try it in the worker now

Comment (klipstein) [inline src/renderer/svg/svg.js:1229]: TypeError: attributes.matrix.clone is not a function :(

Comment (basecode) [inline src/renderer/svg/svg.js:1229]: good point @padolsey. I totally overlooked that one.

Comment (klipstein) [inline src/renderer/svg/svg.js:1229]: @padolsey fixed in f3b7b86eb78b891971ae15fc04672a1a5d1141ee

**Pull request merged**